l10n_es_aeat_sii/models/account_move.py

```diff
@@ -0,0 +1,1447 @@
+    sii_return = fields.Text(string="SII Return", copy=False, readonly=True)
+    sii_header_sent = fields.Text(
+        string="SII last header sent", copy=False, readonly=True,
+    )
+    sii_content_sent = fields.Text(
+        string="SII last content sent", copy=False, readonly=True,
+    )
+    sii_send_error = fields.Text(string="SII Send Error", readonly=True, copy=False,)
+    sii_send_failed = fields.Boolean(
+        string="SII send failed",
+        copy=False,
+        help="Indicates that the last attempt to communicate this invoice to "
+        "the SII has failed. See SII return for details",
+    )
+    sii_refund_type = fields.Selection(
+        selection=[
+            # ('S', 'By substitution'), - Removed as not fully supported
+            ("I", "By differences"),
+        ],
+        string="SII Refund Type",
+        default=lambda self: self._default_sii_refund_type(),
+    )
+    sii_refund_specific_invoice_type = fields.Selection(
+        selection=[
+            ("R1", "Error based on law and Art. 80 One and Two LIVA (R1)"),
+            ("R2", "Art. 80 Three LIVA - Bankruptcy (R2)"),
+            ("R3", "Art. 80 Four LIVA - Bad debt (R3)"),
+            ("R4", "Rest of causes (R4)"),
+        ],
+        help="Fill this field when the refund are one of the specific cases"
+        " of article 80 of LIVA for notifying to SII with the proper"
+        " invoice type.",
+    )
+    sii_account_registration_date = fields.Date(
+        string="SII account registration date",
+        readonly=True,
+        copy=False,
+        help="Indicates the account registration date set at the SII, which "
+        "must be the date when the invoice is recorded in the system and "
+        "is independent of the date of the accounting entry of the "
+        "invoice",
+    )
+    sii_registration_key = fields.Many2one(
+        comodel_name="aeat.sii.mapping.registration.keys",
+        string="SII registration key",
+        default=_default_sii_registration_key,
+        # required=True, This is not set as required here to avoid the
+        # set not null constraint warning
+    )
+    sii_registration_key_additional1 = fields.Many2one(
+        comodel_name="aeat.sii.mapping.registration.keys",
+        string="Additional SII registration key",
+    )
+    sii_registration_key_additional2 = fields.Many2one(
+        comodel_name="aeat.sii.mapping.registration.keys",
+        string="Additional 2 SII registration key",
+    )
+    sii_registration_key_code = fields.Char(
+        related="sii_registration_key.code", readonly=True,
+    )
+    sii_enabled = fields.Boolean(string="Enable SII", compute="_compute_sii_enabled",)
+    sii_property_location = fields.Selection(
+        string="Real property location",
+        copy=False,
+        selection=[
+            (
+                "1",
+                "[1]-Real property with cadastral code located within "
```

Comment:
Typo:
cadastral code -> cadastrial code

Reply:
Creo que es correcto como está:
https://www.wordreference.com/es/translation.asp?tranword=cadastral

Reply:
Ya, si correcto puede ser, simplemente era por coherencia con la definición del siguiente campo:

sii_property_cadastrial_code = fields.Char(string="Real property cadastrial code", size=25, copy=False,)

y es más fácil cambiar la descripción del selector, que el nombre de la columna.

Reply:
@pedrobaeza que opinas?

Reply:
Son campos no demasiado importantes, y aunque es cierto que es incongruente uno con el otro, cambiarlo significa tener que retraducir, así que si podéis aguantar sin que os chirríe mucho, mejor dejarlo como estaba.